app/representers/api/account_representer.rb

```diff
@@ -18,11 +18,18 @@ class Api::AccountRepresenter < Api::BaseRepresenter
   link :image do
     {
       href: api_account_account_image_path(represented),
-      title: represented.image.try(:filename)
-    } if represented.id
+      title: represented.image.filename
+    } if represented.id && represented.image
   end
   embed :image, class: Image, decorator: Api::ImageRepresenter
 
+  link 'create-image' do
+    {
+      href: api_account_account_image_path(represented),
+      title: 'Create an image'
+    } if represented.id && !represented.image
```

Comment:
Once you have an `image` link, you don't need the `create-link`, you can PUT to that and replace it entirely.